--- a/Source/FreeImage/PluginG3.cpp
+++ b/Source/FreeImage/PluginG3.cpp
@@ -132,7 +132,6 @@

 		(*tifin->tif_setupdecode)(tifin);
 		(*tifin->tif_predecode)(tifin, (uint16_t) 0);
-		tifin->tif_row = 0;
 		badfaxlines = 0;
 		badfaxrun = 0;

@@ -152,7 +151,6 @@
 				badrun = 0;
 				_TIFFmemcpy(refbuf, rowbuf, linesize);
 			}
-			tifin->tif_row++;

 			FreeImage_WriteMemory(rowbuf, linesize, 1, memory);
 			row++;
